pkg/asset/internal/templates.go

```diff
@@ -51,6 +49,7 @@ spec:
         - --kubeconfig=/etc/kubernetes/kubeconfig
         - --require-kubeconfig
         - --lock-file=/var/run/lock/kubelet.lock
+        - --containerized
```

Comment:
It is new. It was running because we were using the nsenter workaround which was removed above. The problem with the existing option is that we are using the host mount namespace - which does not contain things like CNI plugins from the hyperkube image.

I had intended to make this change anyway in https://github.com/kubernetes-incubator/bootkube/pull/162 -- as it fixes the issue with port forwarding, but this ends up being required to support the CNI plugins.